### PR TITLE
TOO MUCH! I MESSED UP!

### PR DESCRIPTION
**Pull request by tylerstanczak**
- Add more links to data.js in db folder
- Format db and server for airbnb style
- Fix config file for eslinter
- configure webpack and install dependencies (babel)
- add babel build command to post-deploy script list (pm2) (ask me about this if curious)
- will fix reactDOM.render this week (no component is currently in use

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 });
 const Product = mongoose.model('Product', productSchema);
+
 const connectionString = 'mongodb+srv://tbay:tbay@tbay-pdiop.gcp.mongodb.net/tbay?retryWrites=true&w=majority';
-mongoose.connect(connectionString, {useNewUrlParser: true, useUnifiedTopology: true});
+mongoose.connect(connectionString, { useNewUrlParser: true, useUnifiedTopology: true });
 const db = mongoose.connection;
-Product.insertMany(products);
+
+// Product.insertMany(products);
```
Comment (Xenolithes) [inline database/index.js]: I like how your naming conventions are succinct and are similar to each other.
Line 17 has a bit of commented out code, are you planning on using this later?

File: package.json [hunk 2]
```diff
@@ -16,13 +19,20 @@
     "mongoose": "^5.7.3",
     "mysql": "^2.17.1",
     "nodemon": "^1.19.3",
+    "path": "^0.12.7",
```
Comment (Xenolithes) [inline package.json]: I believe path is build into node.js. Is this a different path node packet?

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: "babel-loader"
+        }
+      }
+    ]
+  }
+};
```
Comment (Xenolithes) [inline webpack.config.js]: You may have to add in rules for css later or it could possibly create a compiling error.

File: package.json (hunk 2 above)
Comment (tylerstanczak) [reply package.json]: so it is... neat!

**Pull request merged**